opennebula/controller.sls

```diff
@@ -57,9 +57,9 @@ one_controller_config_{{ c }}:
   {% else %}
     - contents_pillar: opennebula:lookup:controller:config:{{ c }}:contents
   {% endif %}
-    - user: {{ f.user|default('oneadmin') }}
+    - user: {{ f.user|default('root') }}
```

Comment:
an ls -laR on /var/lib/one shows that per default (under CentOS 7.1) most files are created with owner oneadmin:oneadmin and permission permission 0644 in this folder. 
So if it is the same on all platforms, it's probably good to use these settings per default.
To be honest, I have no Debian, Ubuntu, SuSe, Arch, Gentoo available - and no time - to hv a look at this.